test/Microsoft.ML.Tests/OnnxConversionTest.cs

```diff
@@ -571,9 +572,23 @@ public void KeyToVectorWithBagOnnxConversionTest()
             var onnxTextName = "OneHotBagPipeline.txt";
             var onnxFileName = "OneHotBagPipeline.onnx";
             var onnxTextPath = GetOutputPath(subDir, onnxTextName);
-            var onnxFilePath = GetOutputPath(subDir, onnxFileName);
-            SaveOnnxModel(onnxModel, onnxFilePath, onnxTextPath);
-            CheckEquality(subDir, onnxTextName);
+            var onnxModelPath = GetOutputPath(subDir, onnxFileName);
+            SaveOnnxModel(onnxModel, onnxModelPath, onnxTextPath);
+            // Compare results produced by ML.NET and ONNX's runtime.
+            if (IsOnnxRuntimeSupported())
+            {
+                // Evaluate the saved ONNX model using the data used to train the ML.NET pipeline.
+                var onnxEstimator = mlContext.Transforms.ApplyOnnxModel(onnxModelPath);
+                var onnxTransformer = onnxEstimator.Fit(data);
+                var onnxResult = onnxTransformer.Transform(data);
+                CompareSelectedColumns<float>("Score", "Score", transformedData, onnxResult);
+                CompareSelectedColumns<float>("Probability", "Probability", transformedData, onnxResult);
+                CompareSelectedColumns<bool>("PredictedLabel", "PredictedLabel", transformedData, onnxResult);
+            }
+            else
```

Comment:
Can you please remove the else clause here and in other tests too? (I mean retain the file comparison, but do it always, even if onnx runtime is supported) I meant to do it earlier, but I forgot.
What is happening is that the code in the else clause doesn't get run on our local builds and we forget to update the baselines which leads to build breaks. 
If we always to the file comparison, then the baseline comparison tests will fail in our local builds as well and we will remember to update them.